### PR TITLE
Add Yuan (Terry) Tang to GSoC Mentors

### PR DESCRIPTION
**Pull request by andreyvelich**
We are going to add Yuan to the list of mentors for Training Operator and Katib proposals during GSoC.

/assign @rareddy @tenzen-y @johnugeorge @terrytangyuan

### CODE REVIEW

**Review by tenzen-y — COMMENTED** [review]
@andreyvelich Have you already contacted @terrytangyuan?

/lgtm
/hold
for @terrytangyuan

**Review by terrytangyuan — APPROVED** [review]
Yes, I am aware of this. 

/lgtm

**Comment by andreyvelich** [comment]
> @andreyvelich Have you already contacted @terrytangyuan?
> 
> /lgtm /hold for @terrytangyuan

yes, we got confirmation from @terrytangyuan on Slack.

File: content/en/events/gsoc-2024.md [hunk 1]
```diff
@@ -5,8 +5,7 @@ description = "Google Summer of Code 2024"
 
 ---
 
-Kubeflow Community is excited to announce that we are applying this year to participate in the the [__Google Summer of Code 2024__](https://buildyourfuture.withgoogle.com/programs/summer-of-code)
-
+Kubeflow Community is excited to announce that we are applying this year to participate in the the [**Google Summer of Code 2024**](https://buildyourfuture.withgoogle.com/programs/summer-of-code)
```
Comment (hbelmiro) [inline content/en/events/gsoc-2024.md:8]:
```suggestion
Kubeflow Community is excited to announce that we are applying this year to participate in the [**Google Summer of Code 2024**](https://buildyourfuture.withgoogle.com/programs/summer-of-code)
```

**Comment by rareddy** [comment]
@terrytangyuan has been invited now. 

@tenzen-y please reach out to me on Slack with your email address to be invited as a mentor

**Review by chensun — APPROVED** [review]

**Pull request merged**